packages/backend/src/api/controllers/BlocksController.test.ts

```diff
@@ -15,7 +15,7 @@ describe(BlocksController.name, () => {
       },
     })
     const blocksController = new BlocksController(blockNumberRepository)
-    expect<unknown>(await blocksController.getAllBlocks()).toEqual([
+    expect(await blocksController.getAllBlocks()).toEqual([
```

Comment:
Can you provide the context in PR description why `<unknown>` is no longer needed? (I prefer description because it will stay as a "documentation" for this migration)

Reply:
Done